rs/canister_sandbox/src/replica_controller/sandbox_process_eviction.rs

```diff
@@ -101,17 +129,21 @@ mod tests {
                 id: canister_test_id(i),
                 last_used: now - Duration::from_secs(i),
                 rss: 0.into(),
+                scheduler_priority: AccumulatedPriority::new(0),
             });
         }
         assert_eq!(
             evict(
                 candidates.clone(),
                 0.into(),
-                100,
+                51,
```

Comment:
The functional change is that eviction fn DO NOT evict all idle candidates, but only as much as it needs.